app_spec/test/test.js

```diff
@@ -57,6 +57,19 @@ scenario1.runTape('create_post', async (t, { alice }) => {
   t.equal(result.Ok, "QmY6MfiuhHnQ1kg7RwNZJNUQhwDxTFL45AAPnpJMNPEoxk")
 })
 
+
+scenario1.runTape('delete_post', async (t, { alice }) => {
+
+  const content = "Holo world"
+  const in_reply_to = null
+  const params = { content, in_reply_to }
+  const result = alice.call("blog", "create_post", params)
```

Comment:
I think what you want to do here is:
```suggestion
  const result = await alice.callSync("blog", "create_post", params)
```

But doing that below on `delete_post` might mean that the `waiter` needs to learn about `LinkRemove` action. Right @maackle?

Reply:
Awesome! I will implement that! :) Thank you!